README.rst

```diff
@@ -101,7 +101,11 @@ Copy & paste ...
 
     #!python3
     import blackmamba as bm
+    import console
+
     bm.main()
+    console.clear()
```

Comment:
Please, remove this line and I'll merge it. It clears all messages and that's what we do not want to do. What if console contains errors? Warnings?